docs/README.md

```diff
@@ -1,3 +1,37 @@
+# Installation Prerequisites
+
+1. [Install the latest docker](https://docs.docker.com/get-docker/) and you may want to install [docker desktop - latest](https://docs.docker.com/desktop/)
+2. [Install python 3.9.5 (latest) and](https://www.python.org/downloads/) [configure your environment](https://docs.python.org/3/tutorial/venv.html) to get data-platform source environment ready.
```

Comment:
This overlaps a bit with the in-progress python instructions in [this](https://github.com/usds/justice40-tool/pull/122)  PR -- we should reconcile at some point

Reply:
Yeah, do I need to change the version of python? I think for right now let's just get this merged and we can update later

Reply:
yeah makes sense to merge now then adjust as the other evolves